### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by cvalencia1991**
# In this milestone, I did the following activities:

- Implement Styles to the Based of the template of the project 
- Add new libraries for the style 
- Fix linters Errors

### CODE REVIEW

File: src/Components/booksform.js [hunk 1]
```diff
@@ -24,15 +24,21 @@ const Booksform = () => {
...
       ...book,
-      category: 'book',
     }));
   };
 
   return (
-    <form onSubmit={handleSubmit} className="m-3 d-flex justify-content-center align-items-stretch gap-3 flex-direction-row">
-      <input name="title" type="text" placeholder="Booktitle" onChange={handleChange} required />
+    <form onSubmit={handleSubmit} className="m-3 d-flex justify-content-between align-items-stretch gap-3 flex-direction-row">
+      <input style={{ width: '50%' }} name="title" type="text" placeholder="Booktitle" onChange={handleChange} required />
```
Comment (Ghiftee) [inline src/Components/booksform.js]: - [ ] Kindly remove inline styles as this is not best practice.

File: src/App.css [hunk 2]
```diff
@@ -55,20 +58,212 @@ input {
...
+  background: white;
+  padding-top: 1em;
+  align-items: center;
+  justify-content: space-between;
 }
 
 a {
   font-size: medium;
+  text-decoration: none !important;
+  color: black !important;
```
Comment (Ghiftee) [inline src/App.css]: - [x] I advise that you do not use !important until it is absolutely necessary. In this case, you can do this instead:

``` 
.linkstyles > a {
  font-size: medium;
  text-decoration: none;
  color: black;
}
```

File: src/Navbar.js [hunk 3]
```diff
@@ -14,17 +15,20 @@ const Navbar = () => {
...
+        </div>
       </div>
-      <ul className="stylemenu">
-        {links.map((link) => (
-          <li className="linkstyles" key={link.id}>
-            <Link to={link.path}>{link.text}</Link>
-          </li>
-        ))}
-      </ul>
+      <img style={{ width: '4%' }} src={logo} alt="" />
```
Comment (Ghiftee) [inline src/Navbar.js]: - [ ] Kindly remove inline styling here

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello  @cvalencia1991 👋🏽

Good job working on the project so far! 
✔️ You used the Zeplin colors
✔️ Your app works as expected
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️

_Check the comments under the review._

- [ ] According to the requirements, your styling should look exactly like the one provided so kindly give your app more margin around it like in the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)

### Zeplin design
Screenshot 2022-11-11 at 22 51 23

### Your design
Screenshot 2022-11-11 at 22 51 42

- [x] Please ensure that ```categories``` and ```books```  in your design is spelt in block letters and that ```categories``` is the same color as the one in the zeplin design

### Zeplin
Screenshot 2022-11-11 at 22 51 23

### Your design
Screenshot 2022-11-11 at 22 51 42

- [x] In this section, your design is missing the vertical line between completed circle and the current chapter. Please add it

### Zeplin design
Screenshot 2022-11-11 at 23 10 23

### Your design
Screenshot 2022-11-11 at 23 09 39

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.css (hunk 2 above)
Comment (cvalencia1991) [reply src/App.css]: ok im going to change that

File: src/Navbar.js (hunk 3 above)
Comment (cvalencia1991) [reply src/Navbar.js]: ok i change it

File: src/Components/booksform.js (hunk 1 above)
Comment (cvalencia1991) [reply src/Components/booksform.js]: ok i delete and change it to a class

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @cvalencia1991  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes as requested by the previous reviewer 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cvalencia1991** [comment]
thank you very much 😄

**Pull request merged**